lib/storage/metadata/mongoclient/MongoClientInterface.js

```diff
@@ -2107,34 +2107,55 @@ class MongoClientInterface {
     }
 
     /*
-     * get bucket related information for count items, used by cloudserver
-     * and s3utils
+     * return true if it a system collection, reserved by Mongo for internal use: and which should
+     * not be used.
+     */
+    _isSystemCollection(name) {
+        return name.startsWith('system.');
+    }
```

Comment:
Shouldn't we also prevent creating bucket with such name? I believe that's specific to mongodb so to be handled in Arsenal

Reply:
Somehow, this is already handled I think (i remember somebody tried when we first identified the issue)...
Si, I'd rather keep this separate, since we need to fix the bug for CNES and not sure how we want to fix it (e.g. introduce a way to "escape" names or something)

Reply:
→ https://scality.atlassian.net/browse/ZENKO-4894